src/adContainer/VideoAdContainer.js

```diff
@@ -36,6 +36,8 @@ class VideoAdContainer {
     this.element = createAdContainer();
     this.executionContext = null;
 
+    this.originalVideoElement = Boolean(videoElement);
```

Comment:
would make sense to name it `isOriginalVideoElement` ? Otherwise, with this naming, I would expect it to reference a dom element object?

Reply:
👍